cni/cmd/tc-redirect-tap/main.go

```diff
@@ -376,3 +392,15 @@ type NoPreviousResultError struct{}
 func (e NoPreviousResultError) Error() string {
 	return "no previous result was found, was this plugin chained with a previous one?"
 }
+
+func extractArgs(args, argName string) string {
```

Comment:
Can we add a little documentation to `extractArgs` as well as what format is expected?